velox/functions/lib/aggregates/Compare.h

```diff
@@ -33,4 +33,9 @@ int32_t compare(
     const velox::functions::aggregate::SingleValueAccumulator* accumulator,
     const DecodedVector& decoded,
     vector_size_t index);
-} // namespace facebook::velox::aggregate::prestosql
+
+int32_t compareWithNullAsValue(
```

Comment:
This API needs documentation. 

Since you moved this file to 'lib', the API may need to be modified to make it more generic. Notice  that existing 'compare'-method is Presto-specific. Perhaps, the API should take CompareFlags::NullHandlingMode.

Reply:
Updated, thanks.